injector/pom.xml

```diff
@@ -162,8 +168,28 @@
     </dependency>
     <dependency>
       <groupId>org.ow2.asm</groupId>
-      <artifactId>asm-debug-all</artifactId>
-      <version>5.2</version>
+      <artifactId>asm</artifactId>
+      <version>9.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.ow2.asm</groupId>
+      <artifactId>asm-commons</artifactId>
+      <version>9.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.ow2.asm</groupId>
+      <artifactId>asm-util</artifactId>
+      <version>9.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.ow2.asm</groupId>
+      <artifactId>asm-tree</artifactId>
+      <version>9.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.ow2.asm</groupId>
+      <artifactId>asm-analysis</artifactId>
+      <version>9.0</version>
```

Comment:
Won't all of these come in as dependencies of the base asm dependency?

Reply:
Asm's pom.xml doesn't include transitive dependencies: [1].
However, it seems that only `asm-commons` is actually used.
I removed all other transitive dependencies. Done.

[1] https://repo1.maven.org/maven2/org/ow2/asm/asm/9.0/asm-9.0.pom

Reply:
Thanks @davido, the fix is much appreciated 👍